server/routers/api.ts

```diff
@@ -24,53 +24,188 @@ interface ManifestAccessTicket {
 }
 const manifestAccessTickets: Map<string, ManifestAccessTicket> = new Map();
 
+function applyLogFallbacks(logMessage: types.LogMessageSimple, defaults: Partial<types.LogMessage>) {
+  if (logMessage.entity) {
+    logMessage.entity.detail = {...(defaults.entity?.detail ?? {}), ...(logMessage.entity?.detail ?? {})}; 
+  }
+  logMessage.entity = {...defaults.entity, ...logMessage.entity};
+  logMessage.source = {...defaults.source, ...logMessage.source};
+  logMessage.agent = {...defaults.agent, ...logMessage.agent};
+  return {...defaults, ...logMessage};
+}
+
+function log(context: oak.Context, msg: types.LogMessageSimple, shl?: types.HealthLink) {  
+  let logMessage: types.LogMessage = {
+    version: "2",
```

Comment:
@daniellrgn  this is supposed to be the version of [the logserver schema](https://github.com/uwcirg/logserver/pull/12/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R32), which is `3.0` (see [here](https://github.com/uwcirg/logserver/pull/12/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R11)).

Reply:
Thanks, missed that header. Fixed.